src/batArray.mliv

```diff
@@ -82,8 +82,7 @@ external make : int -> 'a -> 'a array = "caml_make_vect"
     If the value of [x] is a floating-point number, then the maximum
     size is only [Sys.max_array_length / 2].*)
 
-##V<4.2##val make_float : int -> float array
-##V>=4.2##external make_float : int -> float array = "caml_make_float_vect"
+val make_float : int -> float array
```

Comment:
Sorry. I was too casual there to delete the versioning. It appears the `make_float` interface has changed in 4.03 [0]. I've committed a fix to the interface that restores the original behavior. Not sure if it is a good idea to refer to the deprecated `Array.float`. Thoughts?

[0] https://github.com/ocaml/ocaml/blob/trunk/stdlib/array.mli#L55-L62